From 5f3313f958a311ff533b4e122ae20f8269be6a1a Mon Sep 17 00:00:00 2001
From: Aster Boese <asterboese@mailbox.org>
Date: Sat, 1 Mar 2025 20:50:37 -0500
Subject: [PATCH] metainfo: update releases and follow Flathub naming
 convention

This patch is from a PR upstream. https://codeberg.org/dnkl/foot/pulls/1982
It should get pulled in in the next release if accepted.
---
 ...t.metainfo.xml => page.codeberg.dnkl.foot.metainfo.xml | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)
 rename org.codeberg.dnkl.foot.metainfo.xml => page.codeberg.dnkl.foot.metainfo.xml (87%)

diff --git a/org.codeberg.dnkl.foot.metainfo.xml b/page.codeberg.dnkl.foot.metainfo.xml
similarity index 87%
rename from org.codeberg.dnkl.foot.metainfo.xml
rename to page.codeberg.dnkl.foot.metainfo.xml
index 1b7c46a7..ecb978bd 100644
--- a/org.codeberg.dnkl.foot.metainfo.xml
+++ b/page.codeberg.dnkl.foot.metainfo.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-  <id>org.codeberg.dnkl.foot</id>
+  <id>page.codeberg.dnkl.foot</id>
   <metadata_license>MIT</metadata_license>
   <project_license>MIT</project_license>
   <developer_name>dnkl</developer_name>
@@ -34,6 +34,10 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="1.20.2" date="2025-01-18"/>
+    <release version="1.20.1" date="2025-01-03"/>
+    <release version="1.20.0" date="2025-01-01"/>
+    <release version="1.19.0" date="2024-10-23"/>
     <release version="1.18.1" date="2024-08-14"/>
     <release version="1.18.0" date="2024-08-02"/>
     <release version="1.17.2" date="2024-04-17"/>
@@ -50,7 +54,7 @@
     <release version="1.13.1" date="2022-08-31"/>
     <release version="1.13.0" date="2022-08-07"/>
   </releases>
-  <launchable type="desktop-id">org.codeberg.dnkl.foot.desktop</launchable>
+  <launchable type="desktop-id">page.codeberg.dnkl.foot.desktop</launchable>
   <url type="homepage">https://codeberg.org/dnkl/foot</url>
   <url type="bugtracker">https://codeberg.org/dnkl/foot/issues</url>
   <content_rating type="oars-1.1"/>
-- 
2.45.2

